packages/website/pages/_app.js

```diff
@@ -33,6 +33,7 @@ export default function App({ Component, pageProps }) {
       // @ts-ignore
       Sentry.setUser(user)
     }
+    // @ts-ignore
```

Comment:
honestly, we should disable this rule, you shouldn't always need to observe what you call, sometimes you just want to run something once and do []

there are blogs suggesting you move the entire function into useEffect, but I don't personally agree. It can get really big and maybe you want the fn to be transportable.  I think this is just a nonsense warning with no true benefit, IMO.

Reply:
nonblocking, just my opinion